Non-Interactive-Protocol.md

```diff
@@ -11,115 +11,111 @@
+so a party must broadcast the refund transaction after its time-lock expiration.
 
 # General Requirements
 
 A DLC client should have censorship-resistant access to the blockchain, use a
-local fee-estimator and ensure it's well-connected to the tx-relay network.
+local fee-estimator and ensure it is well-connected to the tx-relay network.
 
-We recommend to not react on any mempool event as otherwise it would be a source of
+We recommend clients should register no callbacks on mempool events as doing so may be a source of
 cross-layer mapping. <sup>[Cross-layer Mapping](https://arxiv.org/pdf/2007.00764.pdf)</sup>
 
-We recommend to not produce local non-adaptor signatures for transaction until their broadcast
-is required to avoid their misuage in case of site compromise, assuming an external signer.
+We recommend clients should not produce local (non-adaptor) signatures for CETs until they are about to be broadcast
+in order to avoid their misuse in case of site compromise, assuming an external signer.
 
-Further, we recommend duplicate backups of counterparty signatures to avoid relying on
-counterparty honest behavior in case of primary storage failure.
+Furthermore, we recommend duplicated backups of counter-party signatures be stored to avoid relying on
+honest counter-party behavior in the case of primary storage failure.
 
-Transaction finalization is not implying a mandatory key material access.[ FIXME: Client Key Management. ]
+Transaction finalization does not imply a mandatory key material access. [ FIXME: Client Key Management. ]
 
 # Fee bumping policy
 
-An onchain state machine must take actions based on different timers parameterized by three
-values: `congestion_bump_frequency`/`security_bump_frequency`/`security_point`/`refund_security_point`.
-Their definition and setting recommendations are laid out in this section, their usage is precised
-in following sections.
+An on-chain state machine must take actions based on different clocks parameterized by three
+values: `congestion_bump_frequency`/`security_bump_frequency`/`funding_security_point`/`refund_security_point`.
+These values' definitions and setting recommendations are laid out in this section and their usage is made
+more precise in following sections.
 
-A `security_point` is a client heuristic after which the confirmation of the funding transaction
-is a matter of contract safety, otherwise the user is exposed to a bad-player double-spend if either
-the outcome odds are modified or a signature oracle is released. This point is function of event
-signing time, which may be static or dynamic, we defer point qualification to client.
+A `security_point` is a client-side heuristic time/block-stamp after which the confirmation of a corresponding
+transaction is a matter of contract safety.
+If confirmation does not occur soon after this point, the user can be vulnerable to a malicious double-spend if either
```

Comment:
Is it soon after this point or before this point?

Reply:
all of the spec says when this is reached so I think the current after is correct, lmk if I'm wrong though

Reply:
Yes I think you're right, though I must say it's not super clear when reading this paragraph. But well good enough for now I think.